### DISCUSSION TITLE
[Programmers] Level 2 : 퍼즐 게임 챌린지

### DISCUSSION THREAD

**Discussion by jojaeng2** [BDD-CLUB/challenge-every-week-posting#657] [3. CS(컴퓨터 공학) 관련 글 모음]
최근 알고리즘을 너무 놓은 것 같아서 오랜만에 풀어봤습니다.
앞으로는 반성문을 쓸 것 같으면 알고리즘이라도 꾸준히 풀어보겠습니다  ㅎ
https://blog.naver.com/ds4ouj/223576363677

**Comment by pykido** [comment]
화이팅입니다~!~!~!
